idea/meta-storage/src/common/errors/metadata.ts

```diff
@@ -0,0 +1,9 @@
+import { GearJsonRPCError } from './base';
+
+export class MetadataNotFound extends GearJsonRPCError {
+  name = 'MetadataNotFound';
+}
+
+export class MetaAlreadyExists extends GearJsonRPCError {
+  name = 'MetaAlreadyExists';
+}
```

Comment:
Maybe these errors should be in the `common` package amoung others?

Reply:
In `common` package we contain only `JSON-RPC-ERROR` not the exceptions. In this service, I have just followed and have made the same as in `indexer` service.

For example - https://github.com/gear-tech/gear-js/blob/main/idea/indexer/src/common/errors/program.ts